src/algorithms/cochran.ts

```diff
@@ -0,0 +1,43 @@
+import { CochranResult } from '../types';
+import { standardDeviation } from 'simple-statistics';
+import criticalValueTable from './../grubbs/criticalValueTable';
```

Comment:
This is not the same critical value table that's provided in the task.

The values are different, and also depending on `n`, different critical values apply (which cannot be implemented using Grubbs's critical value table)

Reply:
Yes. That's actually why I asked yesterday if I should create a new critical table.

- [ ] I am creating a new critical table with N and P values based on the submitted excel data.